No -Name Parameter

cadyb opened the pull request:
Test-AzureRmResourceGroupDeployment doesn't have a -Name Parameter. 

Running the code snippet specified results in:
![image](https://user-images.githubusercontent.com/5795530/38148047-ca1184f0-3422-11e8-998f-546f72ef9628.png)


The documentation of Test-AzureRmResourceGroupDeployment doesnt mention a -Name parameter either:
[https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Resources/Commands.Resources/help/Test-AzureRmResourceGroupDeployment.md](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Resources/Commands.Resources/help/Test-AzureRmResourceGroupDeployment.md)

azurecla commented:
Hi __@arambarri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@dragon119 - Please review this public repo PR from @arambarri

arambarri commented:
The changes are already integrated

cadyb committed:
No -Name Parameter

Test-AzureRmResourceGroupDeployment doesn't have a -Name Parameter. 

Running the code snippet specified results in:
Test-AzureRmResourceGroupDeployment : A parameter cannot be found that matches parameter name 'Name'.

The documentation of Test-AzureRmResourceGroupDeployment doesnt mention a -Name parameter either:
https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Resources/Commands.Resources/help/Test-AzureRmResourceGroupDeployment.md
